rllib/utils/replay_buffers/prioritized_replay_buffer.py

```diff
@@ -41,12 +42,18 @@ def __init__(
                 'episodes'. Specifies how experiences are stored.
             alpha: How much prioritization is used
                 (0.0=no prioritization, 1.0=full prioritization).
+            beta: The default value of beta to use. This is useful if beta is
```

Comment:
can you redefine what beta means here, in addition to stating that this is the default value.